src/main/features/response/views/check-and-send.njk

```diff
@@ -118,22 +118,23 @@
         {% endif %}
         {{ row('How this dispute has affected you (optional)', impactOfDisputeText, paths.impactOfDisputePage.evaluateUri({ externalId: claim.externalId })) }}
       {% endif %}
+      {{ tableEnd() }}
 
-      {% if draft.requireDefence() %}
-        {{ row('Your defence', draft.defence.text, paths.defencePage.evaluateUri({ externalId: claim.externalId })) }}
-      {% endif %}
+    {% endif %}
 
+    {% if draft.requireDefence() %}
```

Comment:
Not sure this should be on its own, in its own section. Beside that technically correct.

Reply:
I have done it with Pragnesh and he was happy. I'll ask Tarik if he's happy [at the moment it's better to display it as a separate section than not display it at all]

Reply:
![image](https://user-images.githubusercontent.com/5647337/33725324-b8ab979e-db69-11e7-952a-8fdeee4653df.png)

;)